### PR TITLE
Custom colorpicker

### PR DESCRIPTION
**Pull request by lauble**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

overwhelmed

### CODE REVIEW

**Comment by lauble** [comment]
@Bonkles trying to figure out how to assign you as a reviewer

File: scripts/server.js [hunk 1]
```diff
@@ -4,12 +4,18 @@ import gaze from 'gaze/lib/gaze.js';
...
 
 gaze(['css/**/*.css'], (err, watcher) => {
   watcher.on('all', () => buildCSS());
 });
 
+// // added gaze watcher for all .js files
+// gaze(['**/*.js'], (err, watcher) => {
+//   watcher.on('all', () => buildData());
+// });
+
```
Comment (voscarmv) [inline scripts/server.js:18]: I am still not knowledgeable enough about the code base to review the contents of your code, but my only comment would be to remove some of the leftover comments, such as these, on scripts/server.js :)

Great work, Lauren!

**Review by voscarmv — COMMENTED** [review]
I am still not knowledgeable enough about the code base to review the contents of the code, but my only comment would be to remove some of the leftover comments, such as these, on scripts/server.js

File: scripts/server.js (hunk 1 above)
Comment (Bonkles) [reply scripts/server.js:18]: Great callout @voscarmv - often code reviews are great for catching things like this! Though, in this case the PR is a 'draft' PR that Lauren wanted to push up so I could test it- I expect that further refactors will be required, and at this point it's still okay to have extra comments/other things flying around. 

If this were a non-draft PR, yeah, extra comments, console.logs, etc. would need to be stricken. :)

**Comment by lauble** [comment]
Hey @Bonkles - I need some help with rendering the color picker popup. Here are some screenshots of what I was seeing:

Screenshot 2023-10-25 at 1 54 12 PM

My most recent code shows a small box when I click the icon, and the shadow around the box gets darker with each click:

First click
Screenshot 2023-10-27 at 12 15 22 PM

Second click
Screenshot 2023-10-27 at 12 15 28 PM

Third click
Screenshot 2023-10-27 at 12 15 35 PM

**Comment by Bonkles** [comment]
Closing until further notice! We may come back to this one.

**Pull request closed without merging**